### ISSUE TITLE
How can l get  the order of human key points coordinates and their associated feature vectors

### ISSUE THREAD

**Issue by pinkfloyd06** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#35]
Hello @tensorboy,

Let me first thank you for your clean code.

I have a question related to the order of key points. 

1) Since we have 18 human key points, l would like to know which key point belong to tendon, knee, arm, leg ... ?

2) Is there a way to get a feature vector describing each human key  point ?

3) What does out6= torch.cat([out5_1,out5_2,out1],1) represent ?
It is of dim(185,x,y). What is 185 ?
https://github.com/tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation/blob/master/picture_demo.py#L132

Thank you for you answer

**Comment by Lotayou** [comment]
@pinkfloyd06 
1. You can find an illustration of the openpose skeleton format [here](https://cdn-images-1.medium.com/max/1200/1*oVTetBH6worv5grwvSFkxw.png)
2. The keypoints are stored in `all_peaks`, however there's a problem: It supposed to be an array of shape 18 * #persons * 4, but this is not the case during testing with `ski.jpg`...
3. I don't have a clue either...

**Comment by pinkfloyd06** [comment]
Hi @Lotayou ,

Thank you for your answer.

Please correct me if l am wrong.

we have heatmap_avg of shape [:,:18] where 18 is the number of key points.

We denote by H_t^ j the heatmap for the joint j in frame t  such that H_t^j [ x,y ] is the likelihood of pixel ( x,y ) containing joint j at frame t. Since we deal with probabilities, **sum(heatmap_avg|:,:,18])** is supposed to sum to 1 but it's not the case.

What's wrong ?
